src/gen9_hevc_enc_const_def.h

```diff
@@ -0,0 +1,528 @@
+/*
+ * Copyright © 2017 Intel Corporation
+ *
+ * Permission is hereby granted, free of charge, to any person obtaining a
+ * copy of this software and associated documentation files (the
+ * "Software"), to deal in the Software without restriction, including
+ * without limitation the rights to use, copy, modify, merge, publish,
+ * distribute, sub license, and/or sell copies of the Software, and to
+ * permit persons to whom the Software is furnished to do so, subject to
+ * the following conditions:
+ *
+ * The above copyright notice and this permission notice (including the
+ * next paragraph) shall be included in all copies or substantial portions
+ * of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS
+ * OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF
+ * MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NON-INFRINGEMENT.
+ * IN NO EVENT SHALL PRECISION INSIGHT AND/OR ITS SUPPLIERS BE LIABLE FOR
+ * ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT,
+ * TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE
+ * SOFTWAR
+ *
+ * Authors:
+ *    Chen, Peng <chen.c.peng@intel.com>
+ *
+ */
+
+#ifndef GEN9_HEVC_ENCODER_CONST_DEF_H
+#define GEN9_HEVC_ENCODER_CONST_DEF_H
+
+#define GEN9_HEVC_ENC_MEMETHOD_TABLE 0
+
+static const unsigned int GEN9_HEVC_ME_METHOD[NUM_TU_MODES] =
```

Comment:
I have different idea. If it is not prefixed with static, this variable will be a global variable. It would pollute the whole namespace and it would have some issues if other project also use this variable name as golobal variable. by the way, it should be only used in one file, no duplicated definition is needed here